### DISCUSSION TITLE
bootstrap support

### DISCUSSION THREAD

**Discussion by odion-cloud** [wire-elements/modal#206] [Ideas]
bootstrap support

**Comment by businessdotkom2020** [comment]
yes, we need bootstrap support please?

**Comment by gangi-web** [comment]
Any solutions there, like to have it too?

**Comment by 26kito** [comment]
Did you guys already find out how to do it? I have no idea how to use this component with Bootstrap

**Comment by PhiloNL** [comment]
There is a paid version that ships with support for Bootstrap: https://wire-elements.dev/pro
